app/mutations/MilestoneReport/updateMilestone.ts

```diff
@@ -21,6 +21,7 @@ const mutation = graphql`
         }
         asReportingRequirement {
           ...CollapsibleReport_reportingRequirement
+          reportDueDate
```

Comment:
`reportDueDate` is already in `...CollapsibleReport_reportingRequirement` so I don't know that this solves it?